test/junit/src/test/scala/scala/collection/mutable/MutableListTest.scala

```diff
@@ -1,37 +0,0 @@
-package scala.collection.mutable
-
-import org.junit.runner.RunWith
-import org.junit.runners.JUnit4
-import org.junit.Test
-import org.junit.Assert._
-
-import scala.tools.testing.AssertUtil._
-
-@RunWith(classOf[JUnit4])
-class MutableListTest {
-  
-  // Tests scala/bug#8976
-  @Test def tailIteratorMustTerminateAtLength(): Unit = {
-    val is = MutableList(1,2,3)
-    val tl = is.tail
-    assertEquals(tl.length, tl.iterator.length)
-    is += 5
-    assertEquals(tl.length, tl.iterator.length)
-    assertSameElements(tl, tl.iterator)
-  }
-  @Test def iteratorMustFailEventually(): Unit = assertThrows[NoSuchElementException] {
-    MutableList[Unit]().iterator.next()
-  }
-  // was: Root empty iterator held reference
-  @Test def iteratorMustNotHoldOntoLast(): Unit = {
-    val is = MutableList(Some(1), Some(2))
-    val it = is.iterator
-    val x  = Some(3)
-    is += x
-    assertNotReachable(x, it) {
-      it.next()
-      it.next()
-    }
-    assertTrue(it.isEmpty)
-  }
-}
```

Comment:
Why did you have to remove this file?

Reply:
I removed:
```scala
def assertSameElements[A, B >: A](expected: IterableLike[A, _], actual: GenIterable[B], message: String = ""): Unit
def assertSameElements[A, B >: A](expected: IterableLike[A, _], actual: Iterator[B]): Unit =
```
from `AssertUtil` which were causing ambiguous overloading issues in Dotty. This test relied on them.

Reply:
Sure I can do that. I removed this file since it was testing something that was not part of the strawman collection anyway.

Reply:
Yes, you are right but I prefer keeping these files so that we immediately see what has been migrated to the strawman and what hasn’t.